sorc/machine-setup.sh

```diff
@@ -16,6 +16,8 @@ else
     __ms_shell=sh
 fi
 
+HOMEgfs=${HOMEgfs:-`pwd`/../}
+echo "HOMEgfs=$HOMEgfs"
```

Comment:
please remove.

Reply:
The echo (definitely can go, will remove) or line 19 too? Line 19 supports the change on line 214: 
`. ${HOMEgfs}/versions/build.ver`
This helps the compile_gfs_util_wcoss.sh script interface with machine-setup.sh since it's in a different folder. Happy to go over the issue and consider alternative fixes.

Reply:
echo line removed

Reply:
Any further comments/change requests on this bit?